global.json

```diff
@@ -1,5 +1,6 @@
 {
   "sdk": {
-    "version": "3.0.100-preview9-014004"
+    "version": "3.0.100",
+    "rollForward": "latestFeature"
```

Comment:
Does the `latestFeature` is a valid value here?
The docs only have these values:
`Disable`, `LatestPatch`, `Minor`, `LatestMinor`, `Major`, `LatestMajor`

Reply:
Good question! I'm honestly not sure - see https://github.com/dotnet/designs/pull/71/files#examples-of-usage which is still unclear. I'm okay with `latestMajor` anyway (and we can tweak this any time) - so let's go with that :)

Note the docs for CLI are not the same as `global.json` :)